lib/src/components/NxTextInput/stateful/NxStatefulTextInput.tsx

```diff
@@ -10,7 +10,7 @@ import { initialState, userInput } from '../stateHelpers';
 import { PrivateNxTextInput } from '../NxTextInput';
 
 import { Props, PublicProps, propTypes } from './types';
-export { Props, propTypes } from './types';
+export { Props, PublicProps, propTypes } from './types';
```

Comment:
This is also missing/incorrect in `index.ts` I guess

Reply:
Thanks for catching it, updated here 285bbd59a7ee49a0a3e81ae5b1b202191d6b005a.